opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -364,7 +367,7 @@ def _initialize_components(
     sampler: Optional[Sampler] = None,
     resource_attributes: Optional[Attributes] = None,
     id_generator: IdGenerator = None,
-    logging_enabled: Optional[bool] = None,
+    setup_logging_handler: Optional[bool] = None,
```

Comment:
This will change behavior for custom distro implementors and so I've renamed the param name

Reply:
Much clearer too 👍